nuxeo-webdav-test/src/test/java/org/nuxeo/ecm/webdav/WebDavClientTest.java

```diff
@@ -354,6 +354,11 @@ public void testGetFolderPropertiesAcceptTextXml() throws Exception {
         checkAccept("text/xml");
     }
 
+    @Test
+    public void testGetFolderPropertiesAcceptApplicationXml() throws Exception {
+        checkAccept("application/xml");
```

Comment:
`MediaType.APPLICATION_XML`, `MediaType.TEXT_XML`

Reply:
Yes that's a good idea, but for a future refactoring — I'm keeping the style of the existing code.